spec/features/download_panel_spec.rb

```diff
@@ -76,4 +76,27 @@
       expect(page).to have_css('button[data-sul-embed-toggle="sul-embed-download-panel"]')
     end
   end
+
+  describe 'download file count shows within download button' do
+    it 'has the file count for multiple files in the download panel' do
+      stub_purl_response_with_fixture(multi_file_purl)
+      visit_sandbox
+      fill_in_default_sandbox_form('fw090jw3474')
+      click_button 'Embed'
+      toggle_download_panel
+      within '.sul-i-download-3' do
+        expect(page).to have_css '.sul-embed-download-count[aria-label="number of downloadable files"]', text: 2
+      end
+    end
+    it 'has the file count for files all object resources in the download panel' do
+      stub_purl_response_with_fixture(multi_resource_multi_file_purl)
```

Comment:
a ha!   I guess I will have to make a fixture or something.  Ugh.  Thanks for the tip - I was very puzzled.